x-pack/test/alerting_api_integration/spaces_only/tests/actions/execute_unsecured_action.ts

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import type { SearchTotalHits } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import { ObjectRemover } from '../../../common/lib';
+
+// eslint-disable-next-line import/no-default-export
+export default function createUnsecuredActionTests({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const kibanaServer = getService('kibanaServer');
+  const es = getService('es');
+  const retry = getService('retry');
+
+  describe('execute unsecured action', () => {
+    const objectRemover = new ObjectRemover(supertest);
+
+    // need to wait for kibanaServer to settle ...
+    before(() => {
+      kibanaServer.resolveUrl(`/api/execute_unsecured_action`);
+    });
+
+    after(() => objectRemover.removeAll());
+
+    it('should successfully execute email action', async () => {
+      const testStart = new Date().toISOString();
+      const response = await supertest
```

Comment:
I noticed that the unsecured queuer seems to only allow email connector type: https://github.com/elastic/kibana/blob/c5840b6a2b92201c51160bc53c4cb2b8f5ecf524/x-pack/plugins/actions/server/create_unsecured_execute_function.ts#L23 

But your PR comment indicates you tested with Slack, but you also used `execute` directly.  Maybe we should have a test for an non-email connector and somehow ensure it's queued up?

Reply:
I tried to do a functional test with Slack but I think because we set up simulated external servers for a lot of the 3rd party connectors in the Functional tests I was having trouble getting a success response (although it was being executed directly). I can try to find another one that works.

Reply:
Added a test using a webhook connector in https://github.com/elastic/kibana/pull/178682/commits/c24ebb0c09a766b3f706135a525079ec4a46d0cc

Reply:
Is the limitation of only email connectors ok for the queued executions?  We don't intend these to be queued, we always execute directly?  Or perhaps I'm misunderstanding the email connector check ...

Reply:
We created the ability to queue executions specifically for the notifications service, which only needed email, but maybe since we're not using an allow-list for direct execution, we can remove the allow-list for queuing...

WDYT @mikecote?

Reply:
@ymao1 agreed that we would no longer need this limitation, but not a blocker.

Reply:
I'll do that as a followup since I'll need to change some tests

Reply:
I realized that the ES index action won't work in an unsecured manner so I removed the allowlist and added a denylist that's used by both the direct execute and the schedule functions in this commit: https://github.com/elastic/kibana/pull/178682/commits/3852b6a91f57c175f4ad90781d1ba061d7c96cc2